scripts/overrides/csv_to_es.rb

```diff
@@ -335,7 +335,7 @@ def parse_md_brackets(query)
       # given a markdown style link, parse the part in brackets
       #remove newline character, which confuse regex
       if query
-        query.delete!("\n")
+        query.strip.delete!("\n")
```

Comment:
I think you can use .chomp here, but am not sure if there is a benefit to doing so. @techgique do you know?

Reply:
Yes, it's a little more "Ruby native" to use `chomp!` instead. It catches a couple different line endings whereas this only catches if `\n` is the line ending.

But I looked at the Ruby documentation and tested a couple things here. We actually only need `strip` as that also removes all whitespace characters, including the smaller set that `chomp` removes. So these would be better as just `query.strip!` and updating the comment above them to indicate all leading and trailing whitespace is removed to simplify regex

Reply:
ah ok I misunderstood, let's leave this PR as is then